### PR TITLE
Update simple-profiling.md

### PR DESCRIPTION
**Pull request by kcacciatore**
Identified a missing parameter in the performance update (I was using v11.2.0). Also updated code sample that is runnable from copy/paste, and added link to installing Express.

### CODE REVIEW

**Comment by vsemozhetbyt** [comment]
Thank you! Some style and consistency notes:
1. Long lines in docs and code examples are usually wrapped at 80 characters maximum.
2. In code examples, semicolons are used constantly.

**Comment by vsemozhetbyt** [comment]
cc @nodejs/documentation @nodejs/performance @nodejs/diagnostics @nodejs/v8 (sorry if some mentions are wrong).

File: locale/en/docs/guides/simple-profiling.md [hunk 1]
```diff
@@ -27,10 +27,17 @@ Let's see how the built-in profiler can help provide insight into application
 performance.
 
 To illustrate the use of the tick profiler, we will work with a simple Express
-application. Our application will have two handlers, one for adding new users to
+application (See [here](http://expressjs.com/en/starter/installing.html) for installation instructions). Our application will have two handlers, one for adding new users to
 our system:
 
 ```javascript
+
+const express = require('express', '4.16.4' )
```
Comment (mmarchini) [inline locale/en/docs/guides/simple-profiling.md]: Just to keep the styling of the file.

```suggestion
const express = require('express', '4.16.4' );
```

File: locale/en/docs/guides/simple-profiling.md [hunk 2]
```diff
@@ -27,10 +27,17 @@ Let's see how the built-in profiler can help provide insight into application
...
+const crypto  = require('crypto')
```
Comment (mmarchini) [inline locale/en/docs/guides/simple-profiling.md]:
```suggestion
const crypto  = require('crypto');
```

File: locale/en/docs/guides/simple-profiling.md [hunk 3]
```diff
@@ -27,10 +27,17 @@ Let's see how the built-in profiler can help provide insight into application
...
+const app = express()
```
Comment (mmarchini) [inline locale/en/docs/guides/simple-profiling.md]:
```suggestion
const app = express();
```

File: locale/en/docs/guides/simple-profiling.md [hunk 4]
```diff
@@ -27,10 +27,17 @@ Let's see how the built-in profiler can help provide insight into application
...
+const port = 3000
```
Comment (mmarchini) [inline locale/en/docs/guides/simple-profiling.md]:
```suggestion
const port = 3000;
```

File: locale/en/docs/guides/simple-profiling.md [hunk 5]
```diff
@@ -27,10 +27,17 @@ Let's see how the built-in profiler can help provide insight into application
...
+const users = {}
```
Comment (mmarchini) [inline locale/en/docs/guides/simple-profiling.md]:
```suggestion
const users = {};
```

File: locale/en/docs/guides/simple-profiling.md [hunk 6]
```diff
@@ -72,6 +77,8 @@ app.get('/auth', (req, res) => {
     res.sendStatus(401);
   }
 });
+
+app.listen(port, () => console.log(`profiling example app running on port ${port}`))
```
Comment (mmarchini) [inline locale/en/docs/guides/simple-profiling.md]: Shortened the message to keep it < 80 lines, but feel free to keep the older message and format it differently.

```suggestion
app.listen(port, () => console.log(`Running on port ${port}`));
```

File: locale/en/docs/guides/simple-profiling.md [hunk 7]
```diff
@@ -86,10 +93,15 @@ high latency on requests. We can easily run the app with the built in profiler:
...
+We'll create a new user:
 
 ```
 curl -X GET "http://localhost:8080/newUser?username=matt&password=password"
+```
+
+and put some load on the server using `ab` (ApacheBench):
+
+```
 ab -k -c 20 -n 250 "http://localhost:8080/auth?username=matt&password=password"
```
Comment (ofrobots) [inline locale/en/docs/guides/simple-profiling.md:109]: While we are here. It may be worthwhile to use [autocannon](https://npm.im/autocannon) with `npx` instead. Installing `ab` is much more involved.

**Review by mmarchini — APPROVED** [review]
LGTM with style-nits addressed.

File: locale/en/docs/guides/simple-profiling.md (hunk 7 above)
Comment (ofrobots) [reply locale/en/docs/guides/simple-profiling.md:109]: @kcacciatore thank you for your contribution. Feel free to the the action the above comment if you feel up for it, or leave this for us to do in a follow-on.

File: locale/en/docs/guides/simple-profiling.md [hunk 8]
```diff
@@ -27,10 +27,17 @@ Let's see how the built-in profiler can help provide insight into application
 performance.
 
 To illustrate the use of the tick profiler, we will work with a simple Express
-application. Our application will have two handlers, one for adding new users to
+application (See [here](http://expressjs.com/en/starter/installing.html) for installation instructions). Our application will have two handlers, one for adding new users to
```
Comment (vsemozhetbyt) [inline locale/en/docs/guides/simple-profiling.md]:
```suggestion
application (See [here](http://expressjs.com/en/starter/installing.html) for
installation instructions). Our application will have two handlers, one for
adding new users to
```

File: locale/en/docs/guides/simple-profiling.md [hunk 9]
```diff
@@ -72,6 +77,10 @@ app.get('/auth', (req, res) => {
     res.sendStatus(401);
   }
 });
+
+app.listen(port, () => {
+        console.log(`example app running on port ${port}`)
```
Comment (vsemozhetbyt) [inline locale/en/docs/guides/simple-profiling.md]:
```suggestion
  console.log(`example app running on port ${port}`);
```

File: locale/en/docs/guides/simple-profiling.md [hunk 10]
```diff
@@ -228,13 +242,14 @@ app.get('/auth', (req, res) => {
...
   }
 
-  crypto.pbkdf2(password, users[username].salt, 10000, 512, (err, hash) => {
-    if (users[username].hash.toString() === hash.toString()) {
-      res.sendStatus(200);
-    } else {
-      res.sendStatus(401);
-    }
-  });
+  crypto.pbkdf2(password, users[username].salt, 10000, 512, 'sha512',
```
Comment (vsemozhetbyt) [inline locale/en/docs/guides/simple-profiling.md:247]: Our code usually uses 2 spaces for indentation.

File: locale/en/docs/guides/simple-profiling.md (hunk 7 above)
Comment (kcacciatore) [reply locale/en/docs/guides/simple-profiling.md:109]: @ofrobots - I'll leave it for now - I am totally new to node.js, and typically go learn to debug/profile first when I learn something new. I'm not familiar with autocannon or npx, but will look into it.

File: locale/en/docs/guides/simple-profiling.md [hunk 11]
```diff
@@ -27,10 +27,19 @@ Let's see how the built-in profiler can help provide insight into application
...
 To illustrate the use of the tick profiler, we will work with a simple Express
-application. Our application will have two handlers, one for adding new users to
+application (See [here](http://expressjs.com/en/starter/installing.html) for
+installation instructions). Our application will have two handlers, one for
+adding new users to
 our system:
 
 ```javascript
+
+const express = require('express', '4.16.4' );
```
Comment (Flarna) [inline locale/en/docs/guides/simple-profiling.md]: Why two arguments to `require()`?

Comment (kcacciatore) [reply locale/en/docs/guides/simple-profiling.md]: Just being specific about the version of Express used when I made the change, so if something goes wrong in the future, it provides a clue. I didn't for crypto since it's a core package. Feel free to remove if general practice isn't to include versions of external dependencies...

Comment (fhemberger) [reply locale/en/docs/guides/simple-profiling.md]: Please remove, that's what `package.json` is for. Express method signatures are stable, so there shouldn't be any surprises in future versions.

Comment (kcacciatore) [reply locale/en/docs/guides/simple-profiling.md]: Removed as requested.

File: locale/en/docs/guides/simple-profiling.md [hunk 12]
```diff
@@ -37,7 +37,7 @@ our system:
 const express = require('express');
 const crypto  = require('crypto');
 const app = express();
-const port = 3000;
+const port = 8080;
```
Comment (mcollina) [inline locale/en/docs/guides/simple-profiling.md:40]: why this change?

File: locale/en/docs/guides/simple-profiling.md (hunk 7 above)
Comment (mcollina) [reply locale/en/docs/guides/simple-profiling.md:109]: You can basically achieve the same with:

```
npx autocannon -c 20 -a 250 "http://localhost:8080/auth?username=matt&password=password"
```

This has the benefit of not requiring anything complex to install (especially on Windows).

File: locale/en/docs/guides/simple-profiling.md [hunk 13]
```diff
@@ -86,10 +97,15 @@ high latency on requests. We can easily run the app with the built in profiler:
 NODE_ENV=production node --prof app.js
 ```
 
-and put some load on the server using `ab` (ApacheBench):
+We'll create a new user:
 
 ```
 curl -X GET "http://localhost:8080/newUser?username=matt&password=password"
```
Comment (mcollina) [inline locale/en/docs/guides/simple-profiling.md:103]: I think we should not be recommending to create a new user via a GET.

File: locale/en/docs/guides/simple-profiling.md (hunk 12 above)
Comment (kcacciatore) [reply locale/en/docs/guides/simple-profiling.md:40]: The change just makes the value consistent with the rest of the document

File: locale/en/docs/guides/simple-profiling.md (hunk 13 above)
Comment (kcacciatore) [reply locale/en/docs/guides/simple-profiling.md:103]: Agreed,  but I think this should be included with the suggestion to replace ab with auto cannon, since that's going to require an update to the output and interpretation of the data.
